debian/templates

```diff
@@ -2,7 +2,8 @@ Template: matrix-synapse/server-name
 Type: string
 _Description: Name of the server:
  The name that this homeserver will appear as, to clients and other
- servers via federation. This name should match the SRV record
+ servers via federation. This name should match the "m.server" entry
```

Comment:
I think this is backwards. It should be the name of the server that *hosts* the `.well-known/matrix/server` file (note /server).